api/src/pcapi/core/offers/api.py

```diff
@@ -522,27 +545,43 @@ def update_collective_offer_template(offer_id: int, new_values: dict, user: user
 
 
 def _update_collective_offer(
-    offer: educational_api_offer.AnyCollectiveOffer, new_values: dict, user: users_models.User
+    offer: educational_api_offer.AnyCollectiveOffer,
+    new_values: dict,
+    address_body: offerers_schemas.AddressBodyModel | None,
+    user: users_models.User,
 ) -> list[str]:
     validation.check_validation_status(offer)
     validation.check_contact_request(offer, new_values)
 
+    edit_offer_venue = "offerVenue" in new_values
+    edit_oa = ("locationType" in new_values) or ("locationComment" in new_values) or ("address" in new_values)
```

Comment:
```suggestion
    edit_oa = "locationType" in new_values
```

et imposer au front cet attribut dans le cas ou `locationComment` et/ou `address` est modifié avec oa